src/utils/fields.ts

```diff
@@ -2,7 +2,7 @@ import { DataFrame, PanelData } from '@grafana/data';
 import { DrawStyle, StackingMode } from '@grafana/ui';
```

Comment:
"services" instead of utils? <3

Reply:
<iframe src="https://giphy.com/embed/J336VCs1JC42zGRhjH" width="480" height="339" frameBorder="0" class="giphy-embed" allowFullScreen></iframe>

Reply:
YES

Reply:
Yep, that's something I would've done next. Also inspect all those files in `utils` if they can be arranged better. Feels like too much right now.